src/main/java/edu/ie3/datamodel/utils/validation/ConnectorValidationUtils.java

```diff
@@ -153,6 +153,8 @@ protected static void checkTransformer2WType(Transformer2WTypeInput transformer2
           transformer2WType.getxSc()
         },
         transformer2WType);
+    detectZeroOrPositiveQuantities(
+        new Quantity<?>[] {transformer2WType.getbM()}, transformer2WType);
```

Comment:
Please remove the check for negative quantity (cf. line 143)

Reply:
Done